docs/resources/Channel.md

```diff
@@ -6,36 +6,40 @@ Represents a guild or DM channel within Discord.
+| member?                             | a [thread member](#DOCS_RESOURCES_CHANNEL/thread-member-object) object        | thread member object for the current user, if they have joined the thread, only included on certain API endpoints                                                                               |
+| default_auto_archive_duration?      | integer                                                                       | default duration, copied onto newly created threads, in minutes, threads will stop showing in the channel list after the specified period of inactivity, can be set to: 60, 1440, 4320, 10080   |
+| permissions?                        | string                                                                        | computed permissions for the invoking user in the channel, including overwrites, only included when part of the `resolved` data received on a slash command interaction                         |
+| flags?                              | integer                                                                       | [channel flags](#DOCS_RESOURCES_CHANNEL/channel-object-channel-flags) combined as a [bitfield](https://en.wikipedia.org/wiki/Bit_field)                                                         |
+| total_message_sent?                 | integer                                                                       | number of messages ever sent in a thread, it's similar to `message_count` on message creation, but will not decrement the number when a message is deleted                                      |
+| available_tags?                     | array of [tag](#DOCS_RESOURCES_CHANNEL/forum-tag-object) objects              | the set of tags that can be used in a `GUILD_FORUM` channel                                                                                                                                     |
+| applied_tags?                       | array of snowflakes                                                           | the IDs of the set of tags that have been applied to a thread in a `GUILD_FORUM` channel                                                                                                        |
+| default_reaction_emoji?             | ?[default reaction](#DOCS_RESOURCES_CHANNEL/default-reaction-object) object   | the emoji to show in the add reaction button on a thread in a `GUILD_FORUM` channel                                                                                                             |
+| default_thread_rate_limit_per_user? | integer                                                                       | the initial `rate_limit_per_user` to set on newly created threads in a channel. this field is copied to the thread at creation time and does not live update.                                   |
```

Comment:
can this field only be set on forum channels, or other channels too?
if it can only be set on forum channels, that should be mentioned
if it can be set on other channels, why is it only visible on forum channels in the UI?

Reply:
It only applies to forum channel.